apps/institution-service/cypress/e2e/aggregators.cy.ts

```diff
@@ -0,0 +1,86 @@
+import { Aggregator } from "models/aggregator";
```

Comment:
Add test(s) for potential error paths.

Reply:
Is there an error path other than an invalid token?